wallet/lib/pages/purchase_item/clipper/top_left_bottom_right_clipper.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class TopLeftBottomRightClipper extends CustomClipper<Path> {
+  @override
+  Path getClip(Size size) {
+    final path = Path();
+
+    path.lineTo(0, 18);
```

Comment:
Is there any same clipper in the app?

Reply:
No other clipper was clipping the top right and bottom left corners.